src/pages/timecard/Timecard.spec.jsx

```diff
@@ -80,6 +80,48 @@ describe('Shift spanning mutiple days', () => {
     });
   });
 
+  it('should display start and finish time on timecard when both times have been entered', async () => {
+    renderWithTimecardContext(<Timecard />, {
+      summaryErrors: {},
+      setSummaryErrors: jest.fn(),
+      timeEntries: [
+        {
+          timePeriodTypeId: '00000000-0000-0000-0000-000000000001',
+          startTime: '2022-09-01 01:00:00+00:00',
+          finishTime: '2022-09-03 05:00:00+00:00',
+        },
+      ],
+      setTimeEntries: jest.fn(),
+      timecardDate: '',
+      setTimecardDate: jest.fn(),
+    });
+
+    expect(
+      screen.getByText((content) =>
+        content.includes('01:00 on 1 September to 05:00 on 3 September')
```

Comment:
I think this can conflict with the test name. The test name suggests that it just needs to display the start and finish time when they have both been entered.
But this particular text we'd only want to show when covering multiple days. 
Maybe the test name should be updated.

Reply:
my mistake, that's been changed now